ws4redis/websocket.py

```diff
@@ -198,14 +198,14 @@ def receive(self):
             raise WebSocketError("Connection is already closed")
         try:
             return self.read_message()
-        except UnicodeError:
-            logger.info('websocket.receive: UnicodeError')
+        except UnicodeError as e:
+            logger.info(u'websocket.receive: UnicodeError {}'.format(e))
```

Comment:
Please don't mark strings as unicode.
Instead add ``from __future__ import unicode_literals`` to the top of the file.
This makes a transition to Py3 much easier.